src/views/templates/index.jade

```diff
@@ -0,0 +1,31 @@
+- actorsAndMovies = JSON.parse(actorsAndMovies);
+- var actors = actorsAndMovies.actors;
+
+doctype html
+html
+    head
+        meta(charset='utf-8')
+        meta(http-equiv='X-UA-Compatible', content='IE=edge')
+        meta(name='viewport', content='width=device-width, initial-scale=1')
+        title Hollywood Age Gap
+        meta(name='description', content='The age difference in years between movie love interests.')
+        //script(src='app.js')
```

Comment:
The JS is currently commented out; I will reenable it once I get the search and contribute JS implemented.